docs/Features/XMLFeatures.md

```diff
@@ -128,25 +128,7 @@ See [XML Validation](../Validation.md#xml-validation) for more details.
 
 ## Generate Grammar from XML
 
-### Generate XSD from XML
-
-Using an XML file, an XML Schema/XSD file can be generated from the opening tag in the XML file. With the cursor on the first opening tag, use `Ctrl + .` or `Quick Fix...` and select `Generate foo.xsd and bind with *` to create the file in the the same directory.
-
-![GenerateXSDFromXML](../images/Features/GenerateXSDFromXML.gif)
-
-#### Read more
-
-See [Validation with XSD Grammar](../Validation.md#validation-with-xsd-grammar) for more.
-
-### Generate DTD from XML
-
-Using an XML file, a Document Type Definition/DTD file can be generated from the opening tag in the XML file. With the cursor on the first opening tag, use `Ctrl + .` or `Quick Fix...` and select `Generate foo.dtd and bind with *` to create the file in the the same directory.
-
-![GenerateDTDFromXML](../images/Features/GenerateDTDFromXML.gif)
-
-#### Read more
-
-See [Validation with DTD Grammar](../Validation.md#validation-with-dtd-grammar) for more.
+See [Binding with New Grammar](../BindingWithGrammar.md#quick-fix/codeaction) for info on grammar generation.
```

Comment:
This link doesn't appear to work in the built-in documentation. I get sent to the end of the "Binding With Grammar" page instead of the Quick Fix/ Code Action section